package.json

```diff
@@ -9,6 +9,7 @@
     "@fortawesome/free-solid-svg-icons": "^6.4.2",
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.5.0",
```

Comment:
Could remove axios package as you're using Fetch API

Reply:
Npm uninstalled axios